templates/reference/lf4.html.twig

```diff
@@ -0,0 +1,42 @@
+<head>
+
+</head>
+<body>
+<div style="margin-right: 25%;margin-left: 25%;">
+    <p style="text-align: justify">
+    <h2>DSGVO</h2>
+        Europäische Datenschutz-Grundverordnung:
+        "Verordnung (EU) 2016/679 des Europäischen Parlaments und des Rates vom 27. April 2016
+        zum Schutz natürlicher Personen bei der Verarbeitung personenbezogener Daten,
+        zum freien Datenverkehr und zur Aufhebung der Richtlinie 95/46/EG"
+    <p></p>
+    <h3>Grundsätzlich</h3>
+    <b>Art.1:</b> Diese Verordnung enthält Vorschriften zum Schutz natürlicher Personen bei der
+        Verarbeitung personenbezogener Daten und zum freien Verkehr solcher Daten.
+    <p></p>
+        Diese Verordnung schützt die Grundrechte und Grundfreiheiten natürlicher Personen und
+        insbesondere deren Recht auf Schutz personenbezogener Daten.
+    <p></p>
+    <b>Art 2.:</b> Diese Verordnung gilt für die ganz oder teilweise automatisierte Verarbeitung
+        personenbezogener Daten sowie für die nichtautomatisierte Verarbeitung personenbezogener Daten,
+        die in einem Dateisystem gespeichert sind oder gespeichert werden sollen .
```

Comment:
ist ein leerzeichen vor dem letzten punkt (wird auf der website in der nächsten zeile angezeigt)